### ISSUE TITLE
SSOT missing 1.01, 1.02 & 1.03 resources/branches

### ISSUE THREAD

**Issue by stevieflow** [IATI/IATI-Standard-SSOT#88]
In conversation with @andylolz we looked at the SSOT and wondered why the relevant resources for v 1.01, 1.02, 1.03 were "missing" from this repo

I know some context is that the SSOT was only set up during 1.03 > 1.04. 
Perhaps @Bjwebb or @dalepotter can clarify further?

is there any reason to stop the 1.01>1.03 schema and codelists being in the SSOT?

**Comment by dalepotter** [comment]
This is something that I have often wondered about myself!  Just yesterday @hayfield was looking for a v1.01 schema too.  I'm not even sure where you can find a v1.01 schema, other than an archived website...?

Perhaps @bill-anderson or @wendyrogers might be another one who could shed some light on this, as the set-up of this repo predated my time with IATI.

**Comment by andylolz** [comment]
The older versions of the schema do exist in github branches, e.g. https://github.com/IATI/IATI-Schemas/tree/version-1.01 (and in tags, in fact, although I guess the tags are legacy and the branches are preferred?)

The SSOTs, however, don’t go earlier than 1.04.

**Comment by dalepotter** [comment]
Good spot @andylolz

**Comment by stevieflow** [comment]
Yes, sorry - we know that those version do exist in a repo - just that they are not tagged/linked/pegged in branch of the SSOT.  Correct @Bjwebb ?

**Comment by Bjwebb** [comment]
Correct. http://iatistandard.org/202/developer/ssot/#versions-prior-to-1-04

**Comment by andylolz** [comment]
Thanks @Bjwebb! Would it make sense to retrospectively create these branches, for completeness?
